nncf/api/compression.py

```diff
@@ -224,14 +201,14 @@ def statistics(self, quickly_collected_only: bool = False) -> Dict[str, object]:
             need to keep track of statistics on each training batch/step/iteration.
         :return: A dictionary of printable statistics.
         """
-        return self._loss.statistics(quickly_collected_only)
```

Comment:
Please confirm that all controllers where `self._loss` is used add `self._loss.statistics()` to their statistics. Seems like it is not true now. (For example, see `RBSparsityController`).

Reply:
Done.